html/changelogs/alberyk-moreguns.yml

```diff
@@ -0,0 +1,9 @@
+author: Alberyk, Kyres1, AmoryBlaine
+
+delete-after: True
+
+
+changes: 
+  - imageadd: "Replace a couple of guns sprites that were reference to in real life guns with more futuristic versions."
```

Comment:
Replaces a couple of gun sprites that were references to real life guns, with more futuristic versions.